language-plutus-core/bench/Bench.hs

```diff
@@ -78,3 +105,4 @@ main =
           typeCompare0 = BSL.readFile "test/types/example.plc"
           typeCompare1 = BSL.readFile "bench/example-compare.plc"
           typeCompare = (,) <$> typeCompare0 <*> typeCompare1
+          sampleScript = BSL.readFile "bench/script.plci"
```

Comment:
Comment saying what this is and how you got it.

Perhaps we should split the benchmarks out of `language-plutus-core`? Then we could get e.g. the coordination scripts programmatically, which I think would be much nicer than checking them in.

Reply:
I think that would be a sensible next step, i.e. part of a different PR - I have opened an issue https://github.com/input-output-hk/plutus/issues/1099